src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,20 @@
+package duke.task;
+
+public class Event extends Task {
+
+    protected String at;
+
+    public Event(String description, String at) {
+        super(description);
+        this.at = at;
+    }
+
+    public String getAt() {
+        return at;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString()+ " (at: " + at + ")";
```

Comment:
According to Java Coding Standard, your operators should be preceded and followed by white spaces.